php/includes/language.php

```diff
@@ -91,3 +91,40 @@ function FN_(string $singular, string $plural, ...$replacePairs)
 		wrap_array_keys_in_braces($replacePairs),
 	);
 }
+
+// custom translation framework
+define(
+	"TRANSLATION_TABLE",
```

Comment:
Not sure what's the preferred structure here. A few ideas:
* A constant
* A .json file
In any case, it will have to be read every time.

Reply:
Let's keep it simple for now. As long as we don't read the constant outside of the lib (so with functions to encapsulate it) it's fine with me